### ISSUE TITLE
SoapUI Pro DataGen test step increments on commit

### ISSUE THREAD

**Issue by DyHex** [SmartBear/ready-git-plugin#35]
Not sure if this is a plugin issue or ReadyAPI issue.

I have a few DataGen test steps in a few test cases. 
It's a Number Type property in each that I use for a counter in LoadUI. 

When I save and commit changes through the Git plugin for ReadyAPI 2.5 these DataGen steps somehow increment the property values. Resulting in new uncommitted changes after a commit, the same is the case when you revert changes and then commit again. Which effectively makes you unable to pull changes with the git plugin as you'll always have uncommitted changes. 

Basically the git plugin is unusable if you use DataGen steps in a test case.

**Comment by IlyaAvdeev** [comment]
Hi, DyHex! We know about the issue you faced. It's definitely DataGen issue. Can you please explain why you need to use DataGen (which updates the project file every time you open it) instead of DataSource -> Data Generator? btw, DataGen was marked deprecated more than a year ago, now it's not recommened for use.

**Comment by DyHex** [comment]
Started using soapUI less than a year ago, was not aware DataGen was deprecated, just assumed what's included in the program was supported. Changed to DataSource->Data Generator. Solved the issue, thanks!
